inst/extdata/packages/ntbox.yaml

```diff
@@ -0,0 +1,40 @@
+name : ntbox
+title: "From Getting Biodiversity Data to Evaluating Species Distribution Models in a Friendly GUI Environment"
+version: 0.6.6
+author: "Luis Osorio-Olvera, Andrés Lira-Noriega, Jorge Soberón, Townsend Peterson,
+Manuel Falconi, Rusby G. Contreras-Díaz, Enrique Martínez-Meyer, Vijay Barbe,
+Narayani Barve"
+maintainer: "Luis Osorio-Olvera <luismurao@gmail.com>"
+cran: no
+repository: "https://github.com/luismurao/ntbox"
+description: "NicheToolbox is an R package with a friendly Graphical User Interface (GUI)
+             developed using shiny framework that aims to facilitate the process of building
+             niche models and estimate the species distributions."
+occ_acquisition: yes
```

Comment:
This is an aside from the `sdmverse` review, but you may want to note somehow in the documentation that it is not recommended to use occurrence data directly from `spocc` in studies that are intended for publication, since GBIF datasets should be cited using a DOI. See [link](https://www.gbif.org/citation-guidelines) for more information.

Reply:
Thank you for this recommendation. I just added a note regarding this topic  in the documentation of the function. Please see
https://github.com/luismurao/ntbox/blob/ba483fd35924367a272582a8789aea8ea9803364/R/search_gbif_data.R#L11